packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1167,6 +1197,9 @@ class DropdownButton<T> extends StatefulWidget {
   ///    relative to text direction.
   final AlignmentGeometry alignment;
 
+  /// Defines the borderRadius of the DropdownMenu.
```

Comment:
Please be wary of https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation

This would be a good place exactly what the borderRadius property applies to. Just the button, the menu items, the menu? etc.